app/models/concerns/workflow_extraction.rb

```diff
@@ -212,7 +212,9 @@ def ro_crate
 
   def ro_crate_zip
     ro_crate do |crate|
-      ROCrate::Writer.new(crate).write_zip(ro_crate_path)
+      path = ro_crate_path
+      File.unlink(path) if File.exist?(path)
```

Comment:
why `unlink` rather than `delete`, which I think would be clearer. I'm not sure of the difference between them

Reply:
There's no difference I don't think. I think I was using `unlink` in another code base so it came to mind, but it seems SEEK already makes use of `File.delete` elsewhere so I'll change it to be consistent